SU-SOE/stanford_soe_helper
# SOE-3046: Get Injector 56 into SCSS

# READY FOR REVIEW

# Summary
This adds the Injector file 56 into SCSS.

# Needed By (Date)
- Sprint end

# Criticality
- Collections Injector file 56 into SCSS


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-3046 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-3046)
- edit the injector: `admin/config/development/css-injector/edit/56`
- `drush cc css-js`
- go to: `magazine/article/jeremy-bailenson-taking-grand-tour-latest-virtual-reality` and make sure the changes look good.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3046

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
